cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -40,16 +40,23 @@ func NewCmd(o *Options) *cobra.Command {
 		RunE:    func(_ *cobra.Command, _ []string) error { return cmd.RunWithTimeout() },
 		Aliases: []string{"d"},
 	}
-	cobraCmd.Flags().StringVarP(&o.Source, "source", "s", config.DefaultManagerVersion, `Installation source:
-	- Deploy a specific release  of the lifecycle and module manager: "kyma deploy --source=2.0.0"
-	- Deploy a specific branch of the lifecycle and module manager: "kyma deploy --source=<my-branch-name>"
-	- Deploy a commit (8 characters or more) of the lifecycle and module manager: "kyma deploy --source=34edf09a"
-	- Deploy a pull request, for example "kyma deploy --source=PR-9486"
-	- Deploy the local sources  of the lifecycle and module manager: "kyma deploy --source=local"`)
-	cobraCmd.Flags().StringArrayVarP(&o.Modules, "module", "m", []string{}, `Provide one or more modules to activate after the deployment is finished, for example:
-	- With short-hand notation: "--module name@namespace"
-	- With verbose JSON structure "--module '{"name": "componentName","namespace": "componenNamespace","url": "componentUrl","version": "1.2.3"}'`)
+	cobraCmd.Flags().StringArrayVarP(&o.Kustomizations, "kustomization", "k", []string{}, `Provide one or more kustomizations to deploy. Each occurrence of the flag accepts a URL with an optional reference (commit, branch or release) in the format URL@ref or a local path to the directory of the kustomization file.
+	Defaults to deploying lifecycle-manager and module-manager from github main branch.
```

Comment:
```suggestion
	Defaults to deploying Lifecycle Manager and Module Manager from GitHub main branch.
```

We should either always use a hyphen or never. line 46 and 47 have none.
Also, we should use consistent capitalization.